homework.md

```diff
@@ -20,48 +20,177 @@ CREATE TABLE stations
 );
 ```
 
-1) Write a query that fills out this table. Try your best to pick the correct station name for each ID. You may have to make some manual choices or editing based on the inconsistencies we've found. Do try to pick the correct name for each station ID based on how popular it is in the trip data.
+1. Write a query that fills out this table. Try your best to pick the correct station name for each ID. You may have to make some manual choices or editing based on the inconsistencies we've found. Do try to pick the correct name for each station ID based on how popular it is in the trip data.
 
-Hint: your query will look something like 
+### Question 1 Answer:
 
-``` sql
-INSERT INTO stations (SELECT ... FROM tripe);
+```sql
+CREATE TABLE station_popularity
+(
+    id INT,
+    name VARCHAR(100),
+    popularity integer
+);
+
+INSERT INTO station_popularity
+SELECT Max(from_station_id) as station_id, from_station_name as station_name, count(from_station_name) as popularity
+FROM trips
+GROUP BY station_name;
+
+INSERT INTO station_popularity
```

Comment:
I was trying to use ```with...as``` to create a sub SQL, but I got a few syntax errors so I created two temporary tables to store some of the data that I need.

Reply:
This page may help! <https://www.postgresql.org/docs/9.1/queries-with.html>

For future reference, you can create an actual temp table by specifying `CREATE TEMP TABLE ...`, which is automatically dropped at the end of the session and so you don't need to manually drop it.